crates/astria-composer/src/executor/mod.rs

```diff
@@ -598,7 +607,7 @@ impl Future for SubmitFut {
                             metrics::histogram!(crate::metrics_init::TRANSACTIONS_PER_SUBMISSION)
                                 .record(this.bundle.actions_count() as f64);
 
-                            return Poll::Ready(Ok(*this.nonce + 1));
+                            return Poll::Ready(Ok(this.nonce.wrapping_add(1)));
```

Comment:
Hm, if we ever run into this particular issue then things are going extremely badly. Maybe even just panic here?

Reply:
Done in [44c99af](https://github.com/astriaorg/astria/pull/1081/commits/44c99affc52d66ee3134fa1bbc23f66d21bb632a).